dev/io.openliberty.java.internal_fat/README.txt

```diff
@@ -1,4 +1,4 @@
-The io.openliberty.java.internal_fat FAT runs a sanity check to make sure we are running it against the expected version of Java.  To do this, the typical Open Liberty FAT test setup is used with a WAR file application for testing.  
+The io.openliberty.java.internal_fat FAT runs a validation check to make sure we are running it against the expected version of Java.  To do this, the typical Open Liberty FAT test setup is used with a WAR file application for testing.  
```

Comment:
At the very end of this file, it point people to the "official" source for these tests.  Would you please either add the Java 18 specific information or rewrite it to make it generic?

As an example, from
```
The project for building the WAR file application for this FAT is found in the open-liberty-misc repository, located at https://github.com/OpenLiberty/open-liberty-misc/tree/main/io.openliberty.java.internal_fat_17
```
to
```
The projects for building the WAR file applications for these FATs are found in the open-liberty-misc repository, located at https://github.com/OpenLiberty/open-liberty-misc/tree/main/io.openliberty.java.internal_fat_XX (where XX is the specific version of Java, starting with 17).
```

Reply:
Updated it to make it version agnostic.